EIPS/eip-1109.md

```diff
@@ -0,0 +1,55 @@
+---
+eip: 1109
+title: Remove CALL costs for precompiled contracts
+author: Jordi Baylina (@jbaylina)
+discussions-to: https://github.com/ethereum/EIPs/pull/1109
```

Comment:
Please don't use this PR as a persistent discussion venue - either open an issue for discussion, or a thread on Ethereum Magicians.

Reply:
Changed!